packages/design-system/license-header.js

```diff
@@ -0,0 +1,16 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
```

Comment:
2021

Reply:
Actually most of the files in this PR have the wrong year as well.

Reply:
oh right I meant to look into that. it's enforced with a linting rule (in this package only) so they all have to be the same. it's kind of annoying actually! I think maybe we should deal with it in a separate issue, I'm trying to make as few changes as possible in this PR